examples/all-devices-app/all-devices-common/devices/occupancy-sensor/impl/BUILD.gn

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2025 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/build.gni")
+import("//build_overrides/chip.gni")
+
+source_set("toggling-occupancy-sensor") {
```

Comment:
nit: build path already is `..../occupancy/sensor/impl` so maybe `toggling` is simpler so I can read:

`${chip_root}/examples/.../occupancy/sensor/impl:toggling` instead of
`${chip_root}/examples/.../occupancy/sensor/impl:toggling-occupancy-sensor` 

(shorter, less stutter)